src/kajal/ScannerExamples/Assignment17p5.java

```diff
@@ -0,0 +1,29 @@
+//if no exist in your array return true, else return no not present
+package kajal.ScannerExamples;
+
+import java.util.Scanner;
+
+public class Assignment17p5 {
+
+		boolean isNumberPresent(int[] input, int findNum) {
+			boolean isNumPresent = false;
+			for(int index=0; index<input.length; index++) {
+				if(input[index] == findNum)
+					isNumPresent = true;
```

Comment:
In this case, let say you get number which is you request to check in array on index 2 and still you are iterate arary till length. so instead of this you can write return true on line 12 and on line 14 you can write false. as if number not present in array then it will return false.

let me know if you get comment or not